code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -84,6 +84,7 @@
 	var/del_on_death = 0 //causes mob to be deleted on death, useful for mobs that spawn lootable corpses
 	var/deathmessage = ""
 	var/death_sound = null //The sound played on death
+	var/initial_density //Whether the density of a simple animal starts as 0 or 1. Used for revive().

```

Comment:
Two thing.

You can use initial(variable) to find its initial value. It is a built in proc in BYOND that find its compile time value. So this is unnecessary